<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskComposerD</class>
 <widget class="QDialog" name="TaskComposerD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="buildButton">
       <property name="text">
        <string>4</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="runButton">
       <property name="text">
        <string>5</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stopButton">
       <property name="text">
        <string>T</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="submitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="TestEditor" name="TEditor">
       <attribute name="title">
        <string>Test Cases</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="ChartViewer" name="CViewer">
       <attribute name="title">
        <string>CViewer</string>
       </attribute>
      </widget>
      <widget class="VerdictBrowser" name="VBrowser">
       <attribute name="title">
        <string>Verdict Table</string>
       </attribute>
      </widget>
      <widget class="FileViewer" name="FViewer">
       <attribute name="title">
        <string>FViewer</string>
       </attribute>
      </widget>
      <widget class="GViewer" name="GViewer">
       <attribute name="title">
        <string>GViewer</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartViewer</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.chartviewer.chartviewer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileViewer</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.fileviewer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GViewer</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.gviewer.gviewer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VerdictBrowser</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.verdictbrowser.verdictbrowser</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TestEditor</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.testbrowser.testeditor</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
